### PR TITLE
api文档有误

### PR DESCRIPTION
**Pull request by ITerydh**
### PR types
Bug fixes

### PR changes
Docs

### Describe
api文档这个，代码示例当中是
`l1_loss = paddle.nn.loss.L1Loss(reduction='sum')`
`l1_loss = paddle.nn.loss.L1Loss(reduction='none')`
应该改为：
`l1_loss = paddle.nn.L1Loss(reduction='sum')`
`l1_loss = paddle.nn.L1Loss(reduction='none')`
吧？
错误的文档地址：
[api文档中L1Loss介绍](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/L1Loss_cn.html)

### CODE REVIEW

**Comment by SigureMo** [comment]
该问题已经在 develop 分支（#29932）解决了，可直接查看 develop 版本文档～
